<launch>
    <arg name="checkerboard_squares" default="7"/>
    <arg name="camera_name" default="seek_thermal"/>
    <arg name="min_checkerboard" default="70"/>

    <!-- <param name="/use_sim_time" value="true"/> -->

    <node pkg="thermal_88" type="thermal_calibrator_node" name="thermal_calibrator" output="screen">
        <param name="checkerboard_squares" value="$(arg checkerboard_squares)"/>
        <param name="camera_name" value="$(arg camera_name)"/>
        <param name="min_checkerboard" value="$(arg min_checkerboard)"/>
    </node>

    <node type="rviz" name="calib_rviz" pkg="rviz" args="-d $(find thermal_88)/config/calibrator.rviz"/>
</launch>